javascript/react/redux-i/understanding-pure-impure-functions.md

```diff
@@ -52,12 +52,12 @@ Are the following functions pure or impure?
 
 ```javascript
 function a(x) {
-  x = 'enki'
-  return x
+  x = "enki";
```

Comment:
As mentioned above, this isn't impure since it's not a mutation that exists beyond the lifetime of `a`.

Reply:
changed the question to

```
function a(x) {
  return x * x;
}

function b(x) {
  return Math.random() * x;
}
```